.circleci/config.yml

```diff
@@ -25,37 +25,12 @@ jobs:
           command: ./gradlew
 
       - run:
-          name: Validate Markdown ToCs
+          name: Validate Markdown files
           command: |
-            git grep --files-with-matches 'START doctoc' | xargs doctoc
```

Comment:
ups... it has become quite a mess :no_good:

Reply:
yeah, beyond certain threshold keeping scripts directly in CI config rather obfuscates than clarifies the view